features/flows/rest/pom.xml

```diff
@@ -8,36 +8,58 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.opennms.features.flows</groupId>
   <artifactId>org.opennms.features.flows.rest</artifactId>
-  <packaging>jar</packaging>
+  <packaging>bundle</packaging>
   <name>OpenNMS :: Features :: Flows :: ReST</name>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
+            <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Bundle-Version>${project.version}</Bundle-Version>
+          </instructions>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
   <dependencies>
     <dependency>
       <groupId>org.opennms.core</groupId>
       <artifactId>org.opennms.core.web</artifactId>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.opennms</groupId>
-      <artifactId>opennms-util</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.opennms</groupId>
-      <artifactId>opennms-dao</artifactId>
+      <artifactId>opennms-dao-api</artifactId>
+      <scope>provided</scope>
```

Comment:
This is why I don't like using the Maven plugin that generates the Karaf feature files. By marking these as provided, we omit them from the feature dependencies, but then need to re-declare them if we add a dependency on this (Maven) module.